commons/src/main/java/com/powsybl/commons/report/PowsyblCoreReportResourceBundles.java

```diff
@@ -10,11 +10,14 @@
 /**
  * @author Alice Caron {@literal <alice.caron at rte-france.com>}
  */
-public final class ReportBundleBaseName {
+public final class PowsyblCoreReportResourceBundles {
 
-    public static final String BUNDLE_BASE_NAME = "com.powsybl.commons.reports";
-    public static final String BUNDLE_TEST_BASE_NAME = "i18n.reports";
+    public static final String BASE_NAME = "com.powsybl.commons.reports";
+    public static final String TEST_BASE_NAME = "i18n.reports";
```

Comment:
You should not mix runtime and test configuration. You should create another PowsyblCoreTestReportResourceBundles which contains the 2 bundles like this one and keep a single bundle  BASE_NAME for this one.

Reply:
Sliced in two files, the test one going to powsybl-commons-test module.